airbyte-integrations/bases/base-java-s3/src/main/java/io/airbyte/integrations/destination/s3/jsonl/S3JsonlFormatConfig.java

```diff
@@ -44,15 +83,11 @@ public CompressionType getCompressionType() {
     return compressionType;
   }
 
-  public boolean getFlattenData() {
-    return flattenData;
-  }
-
   @Override
   public String toString() {
```

Comment:
What you've done here is totally fine, but if you don't want to write some of the boilerplate method there is a library called `lombok` which has some annotations you can add to a class to not write all the boiler plate. for example you can add the `@ToString` annotation above the class and it will auto generate this method. Other useful ones are `@Getter` or `@Setter`.